common/events.h

```diff
@@ -51,6 +51,7 @@ enum EVENT : uint32_t {
   EVENT_HELD,
   EVENT_HELD_MEDIUM,
   EVENT_HELD_LONG,
+  EVENT_HELD_XTRA_LONG,
```

Comment:
Please calculate this in your prop instead.
I don't think we need more "HELD" events.
Actually, it looks like you *are* calculating this in the prop, so why add it here??